### PR TITLE
PEP 664: add features

### PR DESCRIPTION
**Pull request by YSCohen**
Added new features under `Features for 3.11` header

### CODE REVIEW

File: pep-0664.rst [hunk 1]
```diff
@@ -81,7 +81,15 @@ Features for 3.11
...
 
- ** Watch this space :) **
+* A ~1.25x speedup from 3.10
+* New -P command line option and PYTHONSAFEPATH environment variable.
+* :pep:`654`, Exception Groups and except*.
+* :pep:`657`, Enhanced error locations in tracebacks.
+* :pep:`646`, Variadic generics.
+* :pep:`655`, Marking individual TypedDict items as required or potentially-missing.
+* :pep:`673`, Self type.
+* :pep:`675`, Arbitrary literal string type.
```
Comment (AA-Turner) [inline pep-0664.rst:91]: This is generally PEP level items, I'm not sure I'd include a command-line option. The below is sorted by PEP number and grouped by normal / typing (as the [beta release announcement](https://discuss.python.org/t/python-3-11-0b2-is-now-available/16111) has been).

```suggestion
* :pep:`654`, Exception Groups and `except*`.
* :pep:`657`, Enhanced error locations in tracebacks.
* :pep:`680`, Support for parsing TOML in the standard library
* A ~1.25x speedup from 3.10

Typing features:
* :pep:`646`, Variadic generics.
* :pep:`655`, Marking individual TypedDict items as required or potentially-missing.
* :pep:`673`, Self type.
* :pep:`675`, Arbitrary literal string type.
* :pep:`681`, Dataclass transforms
```

A

File: pep-0664.rst [hunk 2]
```diff
@@ -81,7 +81,19 @@ Features for 3.11
 
 Some of the notable features of Python 3.11 include:
 
- ** Watch this space :) **
+* :pep:`654`, Exception Groups and ``except*``.
+* :pep:`657`, Enhanced error locations in tracebacks.
+* :pep:`680`, Support for parsing TOML in the standard library
+* A ~1.25x speedup from 3.10
```
Comment (pablogsal) [inline pep-0664.rst:87]: This is a bit simplistic. Please, preserve the original text (mentioning that "Python 3.11 is up to 10-60% faster than Python 3.10. On average, we measured a 1.22x speedup on the standard benchmark suite") as much as possible and link against https://docs.python.org/3.11/whatsnew/3.11.html#faster-cpython.

Comment (YSCohen) [reply pep-0664.rst:87]: @pablogsal isn't it meant to be short snippets? I guess we can add a link

Comment (pablogsal) [reply pep-0664.rst:87]: > @pablogsal isn't it meant to be short snippets

There is no specific requirements, is mainly what the RM decides ;)

Comment (YSCohen) [reply pep-0664.rst:87]: valid. hows this?
> Python 3.11 is up to 10-60% faster than Python 3.10. On average, we measured a 1.25x speedup on the standard benchmark suite. See [Faster CPython](https://docs.python.org/3.11/whatsnew/3.11.html#faster-cpython) for details.

(from [here](https://docs.python.org/3.11/whatsnew/3.11.html#summary-release-highlights))

Comment (pablogsal) [reply pep-0664.rst:87]: Looks good 👍

**Comment by YSCohen** [comment]
when i sign the cla, it redirects to a webpage that just says
> Thank you for authorizing our application, but the CLA must be signed by the users who contributed to the PR. Authors emails are: 78832113+YSCohen@users.noreply.github.com.

in plain text.

**Comment by pablogsal** [comment]
> when i sign the cla, it redirects to a webpage that just says
> 
> >

This may be because you need to set your email in your GitHub account.

CC: @ambv

**Comment by YSCohen** [comment]
> This may be because you need to set your email in your GitHub account.

I have an email. do i need to uncheck this?
image

**Comment by pablogsal** [comment]
> I have an email. do i need to uncheck this?

Very likely, at least while you sign the CLA and meanwhile we try to find a fix for this case (if possible).

**Comment by YSCohen** [comment]
> Very likely, at least while you sign the CLA and meanwhile we try to find a fix for this case (if possible).

I think it may be too late. This PR is "registered" to 78832113+YSCohen@users.noreply.github.com and the CLA gets my regular email, hence "Authors emails are: 78832113+YSCohen@users.noreply.github.com."

**Comment by pablogsal** [comment]
I think for this particular case we can proceed without the CLA (as is mainly copying from text I wrote) but please, ensure you sign the CLA with your email for future contributions. Thanks for the PR!

**Comment by AA-Turner** [comment]
If the email you signed the CLA with is registered with GitHub as a verified email, I'm told it should work -- I've had quite a few problems with the CLA too though, so @ambv is probably best placed to answer / provide guidance.

A

**Comment by YSCohen** [comment]
I'll close this, and @pablogsal can make a new one since he wrote this stuff

File: pep-0664.rst (hunk 2 above)
Comment (AA-Turner) [reply pep-0664.rst:87]:
```suggestion
* Python 3.11 is up to 10-60% faster than Python 3.10. On average, we measured
  a 1.25x speedup on the standard benchmark suite. See `Faster CPython 
  <https://docs.python.org/3.11/whatsnew/3.11.html#faster-cpython>`__
  for details.
```

**Pull request closed without merging**